docs/usage/configuration-options.md

```diff
@@ -441,6 +441,10 @@ You can configure this to `true` if you prefer Renovate to close an existing Dep
 
 ## dependencyDashboardHeader
 
+## dependencyDashboardLabels
+
+Note: the labels will only be updated when the dependency dashboard issue would have been updated for its content and/or title. You are also not allowed to edit the labels: Renovate will restore them on each run.
```

Comment:
```suggestion
The labels only get updated when the Dependency Dashboard issue updates its content and/or title.
You are not allowed to edit the labels, as Renovate bot restores the labels on each run.
```

Not sure if you're not allowed/able to make changes, or if its just pointless to edit the labels, because Renovate throws away your changes?

Reply:
Pointless would be the better term indeed, I've adjusted it